modules/deploy/pages/deployment-option/cloud/gcp-private-service-connect.adoc

```diff
@@ -111,7 +111,7 @@ network_post_body=`cat << EOF
     "cloud_provider": "CLOUD_PROVIDER_GCP",
     "cluster_type": "TYPE_BYOC",
     "name": "<network-name>",
-    "namespace_id": "$NAMESPACE_ID",
+    "resource_group_id": "$RESOURCE_GROUP_ID",
```

Comment:
```suggestion
    "namespace_id": "$NAMESPACE_ID",
```
could you reflect this change in the other v1beta1 calls.

Reply:
done, thanks @sago2k8! We'll update for v1beta2 in a separate PR.